src/twisted/python/__init__.py

```diff
@@ -17,6 +17,13 @@
     "Please use constantly from PyPI instead.",
     "twisted.python", "constants")
 
+
+deprecatedModuleAttribute(
+    Version("Twisted", "NEXT", 0, 0),
```

Comment:
It's documented in Incremental's README: https://github.com/hawkowl/incremental.

If you want to change it (almost certainly not going to happen) you should file a bug there.

(Are we going to adopt incremental into the twisted org, btw @hawkowl ?  I don't love all these factored-out packages living in peoples' personal repos...)